config/puma.rb

```diff
@@ -41,3 +41,4 @@
 
 # Allow puma to be restarted by `bin/rails restart` command.
 plugin :tmp_restart
+worker_timeout 99999999999999999999999999999999999999999999999999
```

Comment:
please remove this. this was done only for debugging. we do not need this as form of code in our PRs